specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/publicIpPrefix.json

```diff
@@ -437,6 +437,10 @@
           "readOnly": true,
           "$ref": "./network.json#/definitions/ProvisioningState",
           "description": "The provisioning state of the public IP prefix resource."
+        },
+        "natGateway": {
+          "$ref": "./network.json#/definitions/NatGateway",
```

Comment:
I think the NatGateway is defined in netGateway.json not in network.json

Reply:
resolved